gcloud/bigtable/happybase/connection.py

```diff
@@ -97,6 +100,16 @@ def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, timeout=None,
             raise ValueError('Protocol cannot be set for gcloud '
                              'HappyBase module')
 
+        if table_prefix is not None:
+            if not isinstance(table_prefix, six.string_types):
```

Comment:
I think I originally based it off of [HappyBase][1] for compatibility.

[1]: https://github.com/wbolster/happybase/blob/9cbd718c10a3089f234f1eac1236b631e1f8e7cd/happybase/connection.py#L114-L119